js/src/edit-free-campaign/setup-free-listings/index.js

```diff
@@ -19,6 +20,39 @@ import FormContent from './form-content';
  * @typedef {import('.~/data/actions').CountryCode} CountryCode
  */
 
+/**
+ * Field names for settings.
+ *
+ * If we are adding a new settings field, it should be added into this array.
+ */
+const settingsFieldNames = [
+	'shipping_rate',
+	'shipping_time',
+	'tax_rate',
+	'website_live',
+	'checkout_process_secure',
+	'payment_methods_visible',
+	'refund_tos_visible',
+	'contact_info_visible',
+];
+
+/**
+ * Get settings object from Form values.
+ *
+ * This method is used to pick out form fields that are specific to settings.
+ * If we are adding a new settings field, it should be added into `settingsFieldNames`.
```

Comment:
💅 
Would it be clearer to point out the **new settings field** as "a new settings field that will be saved via API"?

Reply:
Done in [`ce81aad` (#1317)](https://github.com/woocommerce/google-listings-and-ads/pull/1317/commits/ce81aad3b5a988027d3aa422ed1632ae1955c8e4).